documentation/modules/adding-source-provider.adoc

```diff
@@ -19,8 +19,7 @@ You can add {a-rhv} source provider by using the {project-short} web console.
 
 .Prerequisites
 
-* CA certificate of the {manager}.
-* Apache CA certificate of the {manager}, in case the {manager} Apache CA certificate was replaced with a third-party CA certificate.
+*  {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server.
```

Comment:
the phrasing in #354 (`{manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate`) is better - need to make sure this change won't override it